src/lang/en.json

```diff
@@ -156,6 +156,11 @@
   "FIELD_FLAG_6": "Flag 6",
   "FIELD_FLAG_7": "Flag 7",
   "FIELD_FLAG_8": "Flag 8",
+  "FIELD_SET_TIMER": "Add repeating timer script",
```

Comment:
NIT: This should probably say "Set" rather than "Add" as there's at max one timer executing at any given time.

Reply:
Fixed